src/checkpoints/models/schedule-complexity.model.ts

```diff
@@ -0,0 +1,36 @@
+/* istanbul ignore file */
+
+import { ApiProperty } from '@nestjs/swagger';
+import { BigNumber } from '@polymeshassociation/polymesh-sdk';
+
+import { FromBigNumber } from '~/common/decorators/transformation';
+
+export class ScheduleComplexityModel {
+  @ApiProperty({
+    description: 'ID of the Schedule',
+    type: 'string',
+    example: '123',
+  })
+  @FromBigNumber()
+  readonly id: BigNumber;
+
+  @ApiProperty({
+    description: 'Maximum allowed complexity for the Schedule',
+    type: 'string',
+    example: '1',
```

Comment:
(nit) The maximum example should be bigger than the current
```suggestion
    example: '50',
```